Mage/src/main/java/mage/players/Player.java

```diff
@@ -514,11 +514,27 @@ default int rollDice(Outcome outcome, Ability source, Game game, int numSides) {
 
     int rollDieResult(int sides, Game game);
 
-    default PlanarDieRollResult rollPlanarDie(Outcome outcome, Ability source, Game game) {
-        return rollPlanarDie(outcome, source, game, GameOptions.PLANECHASE_PLANAR_DIE_CHAOS_SIDES, GameOptions.PLANECHASE_PLANAR_DIE_PLANAR_SIDES);
+    default void rollPlanarDie(Ability source, Game game) {
+        switch (rollPlanarDieResult(Outcome.Neutral, source, game)) {
+            case CHAOS_ROLL:
+                game.fireEvent(GameEvent.getEvent(
+                        GameEvent.EventType.ROLLED_PLANESWALK,
```

Comment:
You inverted the EventType and the Roll result in those two fireEvent call.

Reply:
I think that happened because of a merge conflict, this code was pretty old